### DISCUSSION TITLE
How to create default debug signed apk ?

### DISCUSSION THREAD

**Discussion by Joilnen** [axmolengine/axmol#1894] [Q&A]
tried this but didn't works, seems to build only unassigned release 
axmol build -p android -a armv7 -xc '--config','Debug'

**Comment by rh101** [comment]
From [here](https://github.com/axmolengine/axmol/blob/dev/docs/DevSetup.md#supported-options-for-axmol-build):
```
  -xc: additional cmake options: i.e.  -xc '-Dbuild','-DCMAKE_BUILD_TYPE=Release'
  -xb: additional cross build options: i.e. -xb '--config','Release'
```

So try: `axmol build -p android -a armv7 -xb '--config','Debug'`

Also, in case you haven't already done so, ensure you have configured the debug signing config in `app/build.gradle`.

**Comment by Joilnen** [comment]
good worked, thanks.
